resources/schema.edn

```diff
@@ -65,6 +65,10 @@
                                  :progress                 {:type Float}
                                  :attributeNames           {:type    (list String)
                                                             :resolve :resolve/continuous-tree->attributes}
+                                 ;; TODO: are we going to repeat all this common fields in every analysis type
+                                 ;; isn't there a way to abstract it away with union types?
```

Comment:
No entiendo, what is this comment referring to? Is this the discussion we had about having a common Analysis Interface  and deriving Continuos / DiscreteTree etc form it?

Reply:
Yeah exactly. I feel we are repeating some common analysis things and just adding this notes to remember and think about it.